modules/javafx.graphics/src/main/java/javafx/print/JobSettings.java

```diff
@@ -1039,6 +1036,17 @@ public void setPrintQuality(PrintQuality quality) {
 
     /**
      * Property representing an instance of <code>PrintResolution</code>.
+     * A null value is ignored.
+     * <p>
+     * Note that quality and resolution are overlapping concepts.
+     * Therefore a printer may support setting one, or the other but
+     * not both. Applications setting these programmatically should
+     * query both properties and select appropriately from the supported
+     * values. If a printer supports non-standard values, code likely
+     * cannot distinguish the printer's interpretation of these values
+     * and it is safest to stick to selecting a standard value that
+     * matches the requirement.
+     * <p>
```

Comment:
Empty `<p>` tag (you can simply remove this line).

Reply:
done